legion/engine/core/types/meta.hpp

```diff
@@ -254,10 +254,4 @@ namespace legion::core
 
     template<class T, typename... Args>
     inline constexpr bool is_brace_constructible_v = is_brace_constructible<T, Args...>::value;
-
-    template<typename T>
-    struct is_pointer { static const bool value = false; };
```

Comment:
why were these removed instead of made constexpr?

Reply:
nothing uses them, and the STL already has one. I don't know who made this or why they existed in the first place.